framework/components/ema-side-navigation/ema-side-navigation.scss

```diff
@@ -0,0 +1,35 @@
+/**
+* EMA Side Navigation
+*/
+
+.ema-side-navigation {
+  margin: 0;
+}
+
+.ema-side-navigation__link {
+  background-color: map-get($ema-colors, 'primary');
+  color:#fff;
+  display: block;
+  padding: map-get($ecl-spacing, 's');
+  text-decoration: none;
+
+  &:hover {
+    text-decoration: underline;
+  }
+}
+
+.ema-side-navigation__link--depth-1 {
+  background-color: map-get($ema-colors, 'primary');
+}
+
+.ema-side-navigation__link--depth-2 {
+  background-color: map-get($ema-colors, 'blue-1');
+}
+
+.ema-side-navigation__link--depth-3 {
+  background-color: map-get($ema-colors, 'secondary');
```

Comment:
I'd rather use `blue-2` here (even if it is the same color). So if tomorrow they decide to change their secondary color it won't affect this component.